DOCS-1519: Make `GetImage()` changes in the Python SDK (#2346)
viamrobotics/docs: docs/tutorials/services/color-detection-scuttle.md

The program prints a list of robot resources.

> On top of the packages that the code sample snippet imports, add the `random` and the `vlc` packages to the imports, and `pil_to_viam_image` and `viam_to_pil_image` from `viam.media.utils.pil`

Reviewer: Yeah I think this got erroneously copied from the guardian tutorial...
```suggestion
On top of the packages that the code sample snippet imports, import `pil_to_viam_image` and `viam_to_pil_image` from `viam.media.utils.pil`.
```

The top of your code should now look like this:

```python {class="line-numbers linkable-line-numbers"}
import asyncio

> import random
> import vlc

Reviewer: ```suggestion
```

{{< snippet "social.md" >}}

## Full code

```python {class="line-numbers linkable-line-numbers"}
import asyncio
import random

> import vlc

Author: I noticed this was missing and we directed people to import them, but I'm a bit confused as to where they're actually being used?  @npentrel

{{< snippet "social.md" >}}

## Full code

```python {class="line-numbers linkable-line-numbers"}
import asyncio

> import random
> import vlc

Reviewer: ```suggestion
```
